Cloud Application Manager/Deploying Anywhere/deploying-managing-instances.md

```diff
@@ -105,9 +105,13 @@ Select **Always on** if you don’t want to schedule anything. Shutdown powers o
 
 Instance actions (on the instances page or the lifecycle editor) trigger deployment related event scripts from your box. Take these actions to start, stop, terminate an instance, and even perform upgrades or make changes to your live instance.
 
-Some actions are available only after the instance changes state. For example, you can’t forcibly terminate an instance until you’ve terminated it first.
+Some actions are available only after the instance changes state. For example, you can’t forcibly terminate an instance until you’ve terminated it first or you can't force online an instance if it isn't unavailable.
```

Comment:
Hey Julian, I'd better remove this mention from here, and move the screenshot to the **Force Online** section